.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,77 +6,69 @@ labels: "Type: Bug"
+   especially for code snippets, which should look like the following:
+   ```python
+   a = 1
+   b = 2
+   print(a + b)
+   ```
+3. Please be as thorough as possible when describing what went wrong, and what was expected from a
+   correct solution. This might look different depending on the nature of the bug, but more
+   information is always best to ensure you receive the help you need.
 -->
 
 # Add meaningful title here
-<!--
-A clear and concise description of the bug including what happened
-and what you expected to happen.
--->
+<!--The title should clearly define the issue succinctly.-->
+
+<Describe your bug/issue here using as much detail as necessary.>
 
 ## How to reproduce
-<!--
-Describe how another person with no context can recreate this issue.
-It is typically very helpful to reduce the problem as much as possible
-and share a minimum working example. See the note above on including
-code as text rather than screenshots.
--->
+
+<Describe how another person with no context can recreate this issue.>
 
 ## Relevant output
-<!--
-Include any output, plots, or other means of communication here to add context to the problem.
- -->
 
-## GreenHEART version
-<!--
-Share your GreenHEART version and how you installed it. You can print the GreenHEART version from
-a Python REPL or script with these commands:
-```python
-import greenheart
-print(greenheart.__version__)
-```
- -->
+<Include any output, plots, tracebacks, or other means of communication here to add context to the
+problem. All code and full tracebacks should be properly markdown formatted.>
 
 ## System Information
 <!-- Add your information here. -->
- - OS: <e.g. Ubuntu 20.04 or macOS 10.12>
- - Python version: <Result of `python --version`>
- - Relevant library versions, from the results of `pip freeze`, for example:
-   - FLORIS
-   - HOPP
-   - matplotlib
-   - NREL-PySAM
-   - numpy
-   - numexpr
-   - orbit-nrel
-   - pandas
-   - scipy
-   - shapely
+- OS: <macOS 12.4>
+  <!-- e.g. Ubuntu 20.04 or macOS 10.12 -->
+- Python version: <3.10.4>
+  <!-- All OS: `python --version`-->
+- GreenHEART version: <0.1.1>
+  <!--
+  Unix: `pip list --format freeze | grep greenheart`
+  Windows: `pip list --format freeze | findstr greenheart`
+  -->
+  - <Installed from source using an editable installation with developer tools: `pip install -e .[develop]`>
+  - Commit hash: <commit-hash>
+    <!--
+    Unix: pip freeze | grep greenheart | awk -F"git@" '/git@/{print $2}' | awk -F"#egg" '/#egg/{print $1}'
+    -->
+
+### Relevant library versions
+<!--
+Use `pip freeze` to gather the relevant versions, and use the markdown table formatting as
+demonstrated below to replacing all relavant packages and their versions.
+-->
+  
+  | Package | Version |
+  | ------- | ------- |
+  | hopp | <3.1.1> |
+  | numpy | <1.26.4> |
+  | Pyomo | <6.8.0> |
+  | scipy | <1.14.1> |
+  | <another-package> | <version> |
```

Comment:
Maybe change to `<another-relevant-package>` just to be super clear that this is not a list of all environment packages.